signer/src/stacks/contracts.rs

```diff
@@ -987,11 +1070,144 @@ impl AsContractCall for RejectWithdrawalV1 {
     ///    an event on the canonical Stacks blockchain.
     /// 2. That the signer bitmap matches the signer decisions stored in
     ///    this signer's database.
-    async fn validate<C>(&self, _ctx: &C, _req_ctx: &ReqContext) -> Result<(), Error>
+    async fn validate<C>(&self, ctx: &C, req_ctx: &ReqContext) -> Result<(), Error>
     where
         C: Context + Send + Sync,
     {
-        // TODO(255): Add validation implementation
+        eprintln!("check deployer");
+        if self.deployer != req_ctx.deployer {
+            return Err(WithdrawalRejectErrorMsg::DeployerMismatch.into_error(req_ctx, self));
+        }
+
+        // 1. The request exists. Check whether the associated withdrawal request transaction is confirmed on the canonical stacks blockchain. Fail the withdrawal request if it is not on the canonical stacks blockchain.
+        //
+        // 2. The double spend check. Check whether the qualified request ID is in the bitcoin_withdrawals_outputs table, and that any of the associated txids are confirmed on the canonical bitcoin blockchain. Fail the withdrawal request if such a transaction was found.
```

Comment:
Can you wrap these please. Feel free to locally use some tool to do it for you. In vscode I use the [rewrap](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap) extension to do all of that for me.

Reply:
Done